clean_project/clean_file.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
you need this header only if you use unicode characters in the source code, which it seems it's not the case. i think you can remove this